src/Illuminate/Validation/Concerns/ReplacesAttributes.php

```diff
@@ -259,11 +259,11 @@ protected function replaceSize($message, $attribute, $rule, $parameters)
      */
     protected function replaceGt($message, $attribute, $rule, $parameters)
     {
-        if (is_null($value = $this->getValue($parameters[0]))) {
+        if (is_null($value = $this->getValue($parameters[0])) && is_numeric($parameters[0])) {
             return str_replace(':value', $parameters[0], $message);
         }
 
-        return str_replace(':value', $this->getSize($attribute, $value), $message);
+        return str_replace(':value', $this->getSize($attribute, $value ?? $parameters[0]), $message);
```

Comment:
`?:` is only required here

Reply:
I was originally using the `?:`operator in this spot and a few others. But changed it to the `??` just in case the `$value` returned from the `getValue` method turned out to be `0`. Which would cause `$parameters[0]` to be evaluated by the `getSize` method (in this case) as opposed to the `0` the user may have intended. Since the `?:` operator returns falsey for `0` but the `??` does not. 

Apologies if I misunderstood your request. Or if my logic is wrong, thanks for taking the time to review the PR.

Reply:
Ah, I see. Thanks for explaining. :)